src/styles/block/stars.scss

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+  justify-content: center;
+
+  &__item {
+    width: 16px;
+    height: 16px;
+    background-image: url(/src/images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-right: 44px;
```

Comment:
The `margin-right: 44px;` in the `stars__item` class might cause excessive spacing between stars, especially if there are multiple stars in a row. Consider reducing this value or using a more flexible layout approach if alignment issues occur.